examples/more/external-project-add/configure.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mkdir -p ${HOME}/temp/external-project-add/build
+cmake -S . -B ${HOME}/temp/external-project-add/build -DCMAKE_BUILD_TYPE=Release
```

Comment:
Not sure about these commands again, is `${HOME}` relative to the current directory? We want to ensure everything is relative to the repo itself, not a users home directory.

Reply:
I will update it to be as suggested. I added some very specific stuff (specific to my case at least) and forgot the bigger picture, and that it has to work in a more general way.

${HOME} is the users home - not 100% sure it exists by default on all major Linux distros.

To put a bit of a context: I have my sources and do the main part of the dev. in Windows and the Linux builds\tests are inside a virtual machine and I wanted to have the build inside the VM with the sources accessed via a shared folder. This is desired because the Windows shared folder is NTFS - so no sym links when shared objects are created.

Saying it out loud I realize this isn't the way for an examples repo.

Reply:
Ah I see got you, thank you for explaining 👍